djautotask/sync.py

```diff
@@ -416,7 +416,8 @@ class TicketSynchronizer(QueryConditionMixin, Synchronizer):
         'Source': (models.Source, 'source'),
         'IssueType': (models.IssueType, 'issue_type'),
         'SubIssueType': (models.SubIssueType, 'sub_issue_type'),
-        'AssignedResourceRoleID': (models.Role, 'assigned_resource_role'),
+        'AssignedResourceRoleID': (models.Role, 'role'),
```

Comment:
This was left in after removing role

Reply:
Oops. Removed the wrong line. Fixed now.